src/langs/fr.json

```
@@ -79,6 +79,8 @@
   "BOOKMARKED_ITEMS_TITLE": "Signets",
   "FLAG_ITEM_BUTTON": "Signaler",
   "FLAG_ITEM_MODAL_TITLE": "Signaler l'élément",
+  "HIDE_ITEM_SETTING_TITLE": "Cacher pour les lecteurs",
+  "HIDE_ITEM_SETTING_DESCRIPTION": "Si cette option est activée, les lecteurs ne peuvent pas accéder à cet élément ni à son contenu.",
```

Not sure, but I think you should add a comma?
```suggestion
  "HIDE_ITEM_SETTING_DESCRIPTION": "Si cette option est activée, les lecteurs ne peuvent pas accéder à cet élément, ni à son contenu.",
```